### ISSUE TITLE
Hopper and Walker do not train with OpenAI.Baselines

### ISSUE THREAD

**Issue by Sohojoe** [Unity-Technologies/marathon-envs#11]
I've not been able to get Hopper or Walker to train with the ml-agents OpenAI wrapper and baselines. The agents begin training and make some progress but do not converge as they do when using ml-agents's' internal PPO

Test repro: https://github.com/Sohojoe/MarathonEnvsBaselines

Things I have tried:  
* Training strategies: simple_baselines multi-agent, baselines single, baselines multi-agent (MPI). 
* Environments: hopper, walker. 
* Algorithms: PPO2, AC2. 
* Hyperparams (PPO2): default hyper params, train_mujoco hyper params, mimicking ml-agents hyper-params, hyper-params search. 
* Training steps: 1m steps, 5m steps, 16m steps

Based on what I've tried I'm thinking it may be that the reward function (or combination of reward and termination) is overfitting ml-agents's PPO. Next, I will try some variations.

**Comment by Sohojoe** [comment]
I did some more testing and was able to train with OpenAI.Baselines - the problem is with stable_baselines. Will close the issue and open an issue on their site
